@xen-orchestra/backups/_runners/VmsXapi.js

```diff
@@ -19,6 +19,7 @@ const DEFAULT_XAPI_VM_SETTINGS = {
   concurrency: 2,
   copyRetention: 0,
   deleteFirst: false,
+  diskPerVmConcurrency: 99,
```

Comment:
Use `0` for no limit (https://github.com/vatesfr/xen-orchestra/tree/master/%40vates/async-each)